nixos/doc/manual/installation/building-images-via-nixos-rebuild-build-image.chapter.md

```diff
@@ -2,15 +2,26 @@
 
 Nixpkgs contains a variety of modules to build custom images for different virtualization platforms and cloud providers, such as e.g. `amazon-image.nix` and `proxmox-lxc.nix`.
 
-While those can be imported individually, `system.build.images` provides an attribute set mapping variant names to image derivations. Available variants are defined - end extendable - in `image.modules`, an attribute set mapping variant names to a list of NixOS modules.
+While those can be imported directly, `system.build.images` provides an attribute set mapping variant names to image derivations. Available variants are defined - end extendable - in `image.modules`, an attribute set mapping variant names to NixOS modules.
 
-All of those images can be built via both, their `system.build.image` attribute, and the CLI `nixos-rebuild build-image`. To build i.e. an Amazon image from your existing NixOS configuration:
+All of those images can be built via both, their `system.build.image` attribute, as well as the CLI command `nixos-rebuild build-image`. To build i.e. an Amazon image from your existing NixOS configuration, run:
```

Comment:
```suggestion
All of those images can be built via both their `system.build.image` attribute, and the `nixos-rebuild build-image` command.

For example, to build an Amazon image from your existing NixOS configuration, run:
```

Although the example should probably be in an `:::{.example}` block.

Reply:
I applied it to the block on customizing images below, but I don't think rendering this part as a <detail> which is collapsed by default in the manual is a good idea. It's pretty much at the core of this section, not an extra example or such.

Is there a way to not collapse it?

While looking for examples in `nixos/doc/manual` and grepping for example blocks, I only found a single one in "customising packages".

Reply:
> Is there a way to not collapse it?

Yes, change the rendering code. TBH I don't undestand why it was ever made that way. It's the same level of wrong as code comments being low contrast in syntax highlighting: that's very important information, why would you hide it?

Reply:
BTW did you run `rg example **/*.md` or what exactly? Because examples typically get an anchor, so it looks more like `:::{.example #something}`.

Reply:
I mean, I do love rabbit-holes as much as any other nix users, but please forgive me if I don't take a detour to the rendering code of the manual while trying to add 3 new paragraphs to it :sweat_smile: 

The exact command I ran inside the manual dir was:

```
rg ':::\{.example'
```

which did catch blocks with anchors.

Reply:
>  I do love rabbit-holes as much as any other nix users

Yeah I didn't mean you should do it now, but answered the question what can be done :) 

> While looking for examples in `nixos/doc/manual` and grepping for example blocks, I only found a single one in "customising packages".

Ah, it's just NixOS docs, not Nixpkgs. Yes, possible that it's just not particularly well maintained. If people complain "Nix documentation sucks" what they really mean is "NixOS documentation sucks", which is no surprise if barely anyone is working on it.